data/human/marauders.txt

```diff
@@ -563,13 +572,15 @@ ship "Marauder Leviathan" "Marauder Leviathan (Weapons)"
 	add attributes
 		"weapon capacity" 35
 	outfits
-		"Particle Cannon" 4
-		"Heavy Laser Turret" 4
+		"Particle Cannon" 2
+		"Ion Cannon" 2
+		"Heavy Laser Turret" 3
+		"Anti-Missile Turret"
```

Comment:
I feel like this would make more sense as a Chameleon AM as well.

Reply:
what about a bullfrog here and a chameleon for the other one?

Reply:
Yea, for some reason I thought both were Heavies :)